vehicle-control/simulation/Assets/Scripts/DepthCollection.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+using System.Collections;
+
+public class DepthCollection : MonoBehaviour {
+
+	private bool holdingKey = false;
+	private string screenshotName = "screenshot.png";
+	
+	void Start () {
+	}
+	
+	void Update () {
+	
+		if (Input.GetKey (KeyCode.C)) {
+			// For collecting depth data.
+			API.instance.SetOutput("collectKeyPressed", 1);
+			holdingKey = true;
+		}
+		else {
+			if (holdingKey) {
+				screenshotName = "screenshot_"+System.DateTime.Now.ToString("%y%m%d%H%M%s")+".png";
+				Application.CaptureScreenshot("Screenshots/"+screenshotName);
```

Comment:
This doesn't work unless there's a `Screenshots` directory already created in the `simulation` folder. Either create one and commit it, or change this code to work without it.